src/Debugger.php

```diff
@@ -49,8 +49,6 @@ public function startup(object $event): void
             $this->target->addCollector($collector);
             $collector->startup();
         }
-
-        register_shutdown_function([$this, 'shutdown']);
```

Comment:
It's need keep, because run shutdown logic.

Reply:
It's doesn't makes sense except storage flushing.